spec/controllers/tasks_controller_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe TasksController, type: :controller do
+
+    describe "POST#create" do
+        it"returns http success" do
+            post :create, params: { task: {:description => "test1", :is_completed => true}}
+            expect(response).to have_http_status(:success)
+        end
+    end 
```

Comment:
Could you please add a test for the sad path for `create`?

Reply:
done